mage_ai/frontend/components/ContextMenu/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState, useCallback } from 'react';
+
+import FlyoutMenu from '@oracle/components/FlyoutMenu';
+import styled from 'styled-components';
+import { UNIT } from '@oracle/styles/units/spacing';
+import { HEADER_HEIGHT, HEADER_Z_INDEX } from '@components/constants';
+
+export type ContextMenuProps = {
+  children: any;
+  context: ContextMenuEnum;
+};
+
+export enum ContextMenuEnum {
+  FILE_BROWSER,
+}
+
+const ContainerStyle = styled.div<{
+  top?: number;
+  left?: number;
+  right?: number;
+  bottom?: number;
+  width?: number;
+}>`
+  position: fixed;
+  z-index: ${HEADER_Z_INDEX + 100};
+
+  ${props => typeof props.left !== 'undefined' && `
```

Comment:
Does simply using `props.left` instead of `typeof props.left !== 'undefined'` cause issues when `props.left` is 0 or some other issue?

Reply:
Yeah I think that's the reason why undefined is explicitly checked here. I actually copied this `ContainerStyle` from `PopupMenu`, but I assume that's why lol